src/main/kotlin/de/bund/digitalservice/ris/norms/framework/adapter/output/database/dto/Article.kt

```diff
@@ -0,0 +1,25 @@
+package de.bund.digitalservice.ris.norms.framework.adapter.output.database.dto
+
+import java.util.UUID
+import javax.persistence.CascadeType
+import javax.persistence.FetchType
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+import javax.persistence.OneToMany
+import javax.persistence.Table
+
+@Table(name = "articles")
+data class Article(
```

Comment:
Just as a note. With this class name you'll run into `import ... as...` scenarios where we use a no-mapping strategy for the output ports and provide domain entities as parameter. Those entities than need to be mapped to the data transfer object. So in result you'll need both classes to be available in the same scope, so a name collision will happen. If you prefer alias importing, go ahead. This adapter will become quite independent.

Reply:
True, we made use of the alias import option so that we separate the dtos from the entities. This will avoid that the name of the file has something from the package as well. What about if we have this like that and then add it to the confluence page as discussions to settle in our naming conventions? what do you think?

Reply:
Naming convention page/discussion sounds good to me. But in my opinion each adapter can do what ever it wants as long as it is clean code. 🤷🏾